css/style.css

```diff
@@ -0,0 +1,146 @@
+/* GENERAL STYLING */
+
+.crimson-font-color{
+  color: #540002;
+}
+
+.link, .link:visited{
+  color: #666F75;
+  opacity: 0.6;
+}
+
+.link:hover, .link:focus, .link:active {
+  text-decoration: none;
+  opacity: 1;
+  color: #636C72;
+}
+
+.item {
+  padding-bottom: 40px;
+}
+
+.page-head {
+  padding: 50px 0;
+}
+
+/* STYLE CONTAINERS */
+
+.cover .container{
+  padding-top: 200px; 
+}
+
+body, html {
+  height: 100%;
```

Comment:
I like to do `height: 100%; width: 100%;` for html & body to keep percentage sizing consistent. Percentage height doesn't work in CSS unless the element and all of its ancestors have percentage heights. Avoid using percentage heights on elements unless you designed it so that all of its ancestors must have percentage heights.